src/repositories/northstar.js

```diff
@@ -140,6 +140,38 @@ export const updateEmailSubscriptionTopics = async (
   return null;
 };
 
+/**
+ * 
+ * @param {String} id 
+ * @param {[EmailSubscriptionStatus]} emailSubscriptionStatus
+ * @param {Boolean} subscribed 
```

Comment:
I think this can just be a single line:
```suggestion
 * @param {Boolean} emailSubscriptionStatus
```